src/tncdr/stabilizer_mps/pauli_string.py

```diff
@@ -0,0 +1,183 @@
+from copy import copy
+from typing import List
+
+# CONVERSION TABLES
+
+# The two-bit encoding of each Pauli operator is implemented as a python integer. 
+# Pauli strings are also python integers, but consist of 2n bits, where n is the number of qubits.
+single_pauli_to_xz = {'I':0, 'X':1, 'Z':2, 'Y':3}
+xz_to_single_pauli = {0:'I', 1:'X', 2:'Z', 3:'Y'}
+
+phase_to_xz = {'i':1, '-':2, '-i':3}
+xz_to_phase = {0:'', 1:'i', 2:'-', 3:'-i'}
+
+def string_to_xz(description:str)->int:
+    """
+    Converts a Pauli string, provided as an explicit string of 'I', 'X', 'Y' and 'Z' into the corresponding XZ encoding.
```

Comment:
General comment, valid for the following docstrings as well: in principle the description is imperative instead of a description. Citing the [PEP 257 guidelines](https://peps.python.org/pep-0257/):  _The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”._
```suggestion
    Convert a Pauli string, provided as an explicit string of 'I', 'X', 'Y' and 'Z' into the corresponding XZ encoding.
```